frontend/src/app/oe/components/guessing-game/guessing-game.component.ts

```diff
@@ -0,0 +1,93 @@
+import { Component, Input, OnInit, SimpleChanges } from '@angular/core';
+import { Block, BlockTimeStrike } from '../../interfaces/oe-energy.interface';
+import { OeBlocktimeApiService } from '../../services/oe-energy.service';
+import { ToastrService } from 'ngx-toastr';
+
+@Component({
+  selector: 'app-guessing-game',
+  templateUrl: './guessing-game.component.html',
+  styleUrls: ['./guessing-game.component.scss'],
+})
+export class GuessingGameComponent implements OnInit {
+  @Input() fromBlock: Block;
+  @Input() toBlock: Block;
+  @Input() strike: BlockTimeStrike;
+  selectedGuess: string;
+  isLoadingBlock = false;
+
+  constructor(
+    private oeBlocktimeApiService: OeBlocktimeApiService,
+    private toastr: ToastrService
+  ) {}
+
+  ngOnInit(): void {
+    // if (this.strike) {
```

Comment:
imho, commented out code should not be merged: after a while we wouldn't know why is it here and if it's still correct or not

Reply:
Done removed